app/tests/test_user.py

```diff
@@ -0,0 +1,179 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from app.models import User
+from app.serializers import UserSerializer
+
+class UserTestCase(TestCase):
+    fixtures = ('user',)
+    SERVER_NAME = 'http://gucci.com/api'
+
+    def setUp(self):
+        self.client = APIClient()
+        self.user = User.objects.get(email='eduard.oancea@algotech.solutions')
+        self.client.force_authenticate(user=self.user)
+
+    def test_get_one(self):
+        config = { 'url': '/users/1/', 'pk': 1}
```

Comment:
What happens when there is an invalid id in url (for get_one, or update, or destroy a.k.a delete)?

Cases:
- invalid id is number (Ex: /users/243424/)
- invalid id is string (Ex: /users/invalid_id/)